File: app/src/main/java/vn/com/basc/inco/model/Company.java
package vn.com.basc.inco.model;

/**
 * Created by SONY on 7/3/2016.
 */
public class Company {
    private String ClientID;
    private String ClientName;
    private String ClientCode;
    private String Logo;
    private String Description;
    private String DateCreated;
    private String DateExpired;
    private String BuildNumber;
    private String Status;
    private String url_ios;
    private String url_android;
    private String msg;
    private String android_ver;

    public String getAndroid_ver() {
        return android_ver;
    }

    public String getMsg() {
        return msg;
    }

    public String getClientID() {
        return ClientID;
    }

    public String getClientName() {
        return ClientName;
    }

    public String getClientCode() {
        return ClientCode;
    }

    public String getLogo() {
        return Logo;
    }

    public void setClientID(String clientID) {
        ClientID = clientID;
    }

    public void setClientName(String clientName) {
        ClientName = clientName;
    }

    public void setClientCode(String clientCode) {
        ClientCode = clientCode;
    }

    public void setLogo(String logo) {
        Logo = logo;
    }

    public String getDescription() {
        return Description;
    }

    public String getDateCreated() {
        return DateCreated;
    }

    public String getDateExpired() {
        return DateExpired;
    }

    public String getBuildNumber() {
        return BuildNumber;
    }

    public String getStatus() {
        return Status;
    }

    public String getUrl_ios() {
        return url_ios;
    }

    public String getUrl_android() {
        return url_android;
    }

    @Override
    public String toString() {
        return ClientCode;
    }
}

File: Inco/Company.swift
//
//  Company.swift
//  Inco
//
//  Created by admin on 8/1/16.
//  Copyright © 2016 Ban Sac. All rights reserved.
//

import Foundation
class Company {
    var ClientID:String?
    var ClientName:String?
    var ClientCode:String?
    var Logo:String?
    var Description:String?
    var DateCreated:String?
    var DateExpired:String?
    var BuildNumber:String?
    var Status:String?
    var url_ios:String?
    var url_android:String?
    var msg:String?
    var android_ver:String?
    init(data:NSDictionary){
        self.ClientID = data.valueForKey("ClientID") as? String
        self.ClientName = data.valueForKey("ClientName") as? String
        self.ClientCode = data.valueForKey("ClientCode") as? String
        self.Logo = data.valueForKey("Logo") as? String
        self.Description = data.valueForKey("Description") as? String
        self.DateCreated = data.valueForKey("DateCreated") as? String
        self.DateExpired = data.valueForKey("DateExpired") as? String
        self.BuildNumber = data.valueForKey("BuildNumber") as? String
        self.Status = data.valueForKey("Status") as? String
        self.url_ios = data.valueForKey("url_ios") as? String
        self.url_android = data.valueForKey("url_android") as? String
        self.msg = data.valueForKey("msg") as? String
        self.android_ver = data.valueForKey("android_ver") as? String



    }
}